torchvision/models/swin_transformer.py

```diff
@@ -324,7 +342,9 @@ def __init__(
         # split image into non-overlapping patches
         layers.append(
             nn.Sequential(
-                nn.Conv2d(3, embed_dim, kernel_size=patch_size, stride=patch_size),
+                nn.Conv2d(
+                    3, embed_dim, kernel_size=(patch_size[0], patch_size[1]), stride=(patch_size[0], patch_size[1])
```

Comment:
Nit:
```suggestion
                    3, embed_dim, kernel_size=tuple(patch_size), stride=tuple(patch_size)
```

Reply:
This is actually what I did first, but unfortunately mypy complaints :(
```
swin_transformer.py:346: error: Argument "kernel_size" to "Conv2d" has incompatible type "Tuple[int, ...]"; expected "Union[int, Tuple[int, int]]"  [arg-type]
                        3, embed_dim, kernel_size=tuple(patch_size), stride=(patch_size[0], patch_size[1])
```